Altis_Life.Altis/stringtable.xml

```diff
@@ -2893,7 +2893,7 @@
             <Spanish>Perdonar</Spanish>
             <Russian></Russian>
             <German>Erlassen</German>
-            <French>Supprimer</French>
+            <French>Pardonner</French>
```

Comment:
Since we're deleting something from the user's records, "Supprimer" should do the trick instead of "Pardonner". That's another meaning.

Reply:
"Pardonner" : is the most appropriate in this case.

Reply:
Can be debated too.

Reply:
Still not good with that point. "Pardonner" or "Excuser" don't apply there, because like I said before we're removing something from his record. Even the police (gendarmerie, same) is saying that.